### ISSUE TITLE
Window buttons are shown twice

### ISSUE THREAD

**Issue by Filbuntu** [hardpixel/unite-shell#126]
This extension is great. I only have a minor cosmetic issue. The window buttons set is shown twice. Only the second set would be responsive to clicks, though. Interestingly the first set of buttons sometimes disappears but reappears after a certain time, e.g. maybe after returning from a locked screen.
![Screenshot from 2019-09-11 10-21-19](https://user-images.githubusercontent.com/12456480/64663322-9d7d2f80-d475-11e9-9c63-992f90dcf237.png)

I tried to hide them according to [this answer](https://askubuntu.com/a/948321/30631): 
`gsettings set org.gnome.desktop.wm.preferences button-layout :`
But it would hide both sets of button. 
Latest officially available Unite(-shell) Gnome extension is installed on Ubuntu 19.04.
Can anybody help with this?

Here are my settings:
![Screenshot from 2019-09-11 10-23-32](https://user-images.githubusercontent.com/12456480/64663429-f3ea6e00-d475-11e9-9bb1-1a413c0c8653.png)

**Comment by jonian** [comment]
Hi @Filbuntu, sorry for the late response.

I can't replicate the issue. Please try the latest version.

If the issue is not resolved, provide the output of `journalctl /usr/bin/gnome-shell`.

**Comment by Filbuntu** [comment]
Hi @jonian! Thanks a lot for your answer! I did not expect a quick answer as it is not essential problem, just a cosmetic one.
Thank you, too, for the tip to look at the output of `journalctl /usr/bin/gnome-shell.`
I found a recurrent interesting extension error:
`JS ERROR: TypeError: global.screen is undefined
                                           getWindow@/home/../.local/share/gnome-shell/extensions/pixel-saver@deadalnix.me/util.js:9:1
                                           updateVisibility@/home/.../.local/share/gnome-shell/extensions/pixel-saver@deadalnix.me/buttons.js:209:13
`
Which led me to the [bug report "global.screen is undefined error on gnome-shell 3.30 (running on antergos) · Issue #178 · pixel-saver/pixel-saver"](https://github.com/pixel-saver/pixel-saver/issues/178).

I had an red "Error" message beside the Pixel Saver extension entry in the list [Installed Extensions - GNOME Shell Extensions ](https://extensions.gnome.org/local/) and assumed the[ Pixel Saver extension](https://extensions.gnome.org/extension/723/pixel-saver/) was not active.

**I deleted the Pixel Saver extension** and it looks like the second button group has disappeared, wonderful!
Thanks again for your help!
—
